crates/core/src/db/datastore/locking_tx_datastore/mut_tx.rs

```diff
@@ -113,10 +113,12 @@ impl MutTxId {
             table_type: table_schema.table_type,
             table_access: table_schema.table_access,
         };
+        println!("TABLE ROW: {:?}", row);
```

Comment:
You may forgot to remove println and subsequents?

Reply:
Thanks